app/grandchallenge/ai_website/templates/ai_website/base.html

```diff
@@ -0,0 +1,91 @@
+{% extends "base.html" %}
+{% load static %}
+{% load ai_website_tags %}
+
+{% block style %}
+  <link rel="stylesheet"
+      href="{% static 'vendor/css/bootstrap.min.css' %}">
+  <link rel="stylesheet" type="text/css"
+      href="https://cdn.datatables.net/v/bs4/dt-1.10.18/b-1.5.4/b-colvis-1.5.4/datatables.min.css"
+      crossorigin="anonymous">
+  <link href="{% static 'vendor/fa/css/all.css' %}" rel="stylesheet">
+
+  <link rel="stylesheet" href="{% static 'ai_website/css/main.css' %}">
+  <link rel="icon" href="{% static 'ai_website/images/logo_16.ico' %}">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Raleway">
+  <style>
+    body,h1,h2,h3,h4,h5,h6 {font-family: "Raleway", sans-serif}
+  </style>
```

Comment:
Can we use {{ block.super }} in any way here?

Reply:
We could add an extra block in `style.html`, but that does not make it much cleaner in my opinion. Also, when I tried that locally for some reason some of the styling broke, so I'd rather keep it like this for now and possibly debug the issue later. I did remove `datatables.js`, as it's not used at all in the products app.

Reply:
:+1: